src/Cake.Common/Tools/OpenCover/OpenCoverSettingsExtensions.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.OpenCover
+{
+    /// <summary>
+    /// Contains extensions for <see cref="OpenCoverSettings"/>.
+    /// </summary>
+    public static class OpenCoverSettingsExtensions
+    {
+        /// <summary>
+        /// Adds the filter.
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <param name="filter">The filter.</param>
+        /// <returns>The same <see cref="OpenCoverSettings"/> instance so that multiple calls can be chained.</returns>
+        public static OpenCoverSettings WithFilter(this OpenCoverSettings settings, string filter)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException("settings");
+            }
+            settings.Filters.Add(filter);
+            return settings;
+        }
+
+        /// <summary>
+        /// Exclude a class or method by filter 
```

Comment:
@patriksvensson is there a need to break the Summary onto two lines?  Won't this result in two separate <p> tags when the documentation is created?